policyengine_us/parameters/gov/irs/ald/alimony_expense/divorce_year_threshold.yaml

```
@@ -0,0 +1,9 @@
+description: The IRS allows for an above the line deduction for alimony expenses incurred before the following divorce year.
```

Make this a scale parameter: true until 2019, then false, then true again in 2026
```suggestion
description: The IRS allows for an above the line deduction for alimony expenses for divorces that occurred in these years.
```